### PR TITLE
Add org.gnome.Hitori

### PR DESCRIPTION
**Pull request by pwithnall**
This uses the 3.22.4 release (which is the latest stable release; we
don’t have a 3.24 release series yet).

Signed-off-by: Philip Withnall <withnall@endlessm.com>

### CODE REVIEW

File: org.gnome.Hitori.json [hunk 1]
```diff
@@ -0,0 +1,47 @@
+{
+  "app-id": "org.gnome.Hitori",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "3.24",
```
Comment (cosimoc) [inline org.gnome.Hitori.json]: I wonder if you could use 3.26 instead.

**Comment by cosimoc** [comment]
Test build: https://flathub.org/builds/#/buildrequests/3004?redirect_to_build=true

**Comment by cosimoc** [comment]
Built fine, so I think this is good to go.

**Comment by TingPing** [comment]
Repository has been created: https://github.com/flathub/org.gnome.Hitori

File: org.gnome.Hitori.json (hunk 1 above)
Comment (pwithnall) [reply org.gnome.Hitori.json]: I’ve changed it to 3.26 now. Not necessary, but does mean we don’t force the user to download the 3.24 runtime unnecessarily, given that most other apps use 3.26.

**Pull request closed without merging**